lib/usd/schemas/plugInfo.json.in

```diff
@@ -13,7 +13,8 @@
                         "autoGenerated": true, 
                         "bases": [
                             "UsdGeomXformable"
-                        ]
+                        ],
+                        "schemaKind": "concreteTyped"
```

Comment:
Why `schemaType` wasn't here before? All other changes are basically a rename, but here we have an addition.

Reply:
This was changed so that we no longer need to load all plugins and generatedSchema.usda files and process all the UsdPrimDefinitions just to query about the kind of a schema type. It's more in line with our "pay for what you use" policy in USD. We've been moving a lot more information about schema types into plugInfo metadata over the past year